packages/app/src/cli/services/extensions/bundle.ts

```diff
@@ -142,3 +144,16 @@ function isGraphqlPackageAvailable(): boolean {
     return false
   }
 }
+
+// This will probably be on the extension instance after Isaac's refactor
+export function getBundleExtensionStdIn({configuration, directory, entrySourceFilePath}: UIExtension) {
+  if (configuration.type === 'ui_extension') {
+    const extensionPoints = configuration.extensionPoints as NewExtensionPointType
+
+    return extensionPoints.map(({module}) => `import '${module}';`).join('\n')
+  }
+
+  const relativeImportPath = (entrySourceFilePath as string).replace(directory, '')
```

Comment:
Again, I didn't invest in type safety here since:

1. The plan is this code will never get merged into master.
2. After Isaac's refactor, and migrating existing extensions to the `ui_extension` type, this should be much simpler